week-3/Homework/mandatory/2-bush-berries.js

```diff
@@ -24,29 +24,54 @@ will have a remainder of 0 i.e. they are even numbers*/
 array.every((value) => {return (value % 2 == 0)}); /* this will return false as not ALL 
 values will have a remainder of 0 i.e. there are some odd numbers in the array too*/
 
+// if (arr.some((color) => {return (color === "neon")}); 
+
 /* + + + + + + + + + + + + + + */
 
 /* Now try to complete the exercise */
 
-function bushChecker() {
+let bushColours = ["pink", "pink", "pink", "pink"]
+
+function bushChecker(arr) {
+
+// checks whether an element is neon
+const equalNeon = (element) => element === "neon";
+
+let notPink = (arr.some(equalNeon));
+console.log(notPink)
 
+// expected output: false
+  
+if (notPink === true) 
+  {
+  return console.log("this is toxic")
+  
+ } 
+ else {
+   return console.log("safe to eat")
+   // expected output: safe to eat
+ }
```

Comment:
styling review

Hey! Just to make it easier for developers of you team in the future. Recommend using `Prettier` extensions - don't be so drawn to keeping questions looking the same. If you can improve on something, please do.

It'll help you in creating more beautiful code! 👓